source-code-git/fs/src/memory/implementation.ts

```diff
@@ -0,0 +1,121 @@
+import type { Filesystem } from "../schema.js"
+
+type FileData = string
+type Directory = Map<string, Inode>
+type Inode = FileData | Directory
+
+export class memoryFs implements Filesystem {
```

Comment:
Classes should be CamelCase. Please rename `memoryFs` to `MemoryFs` :)

Reply:
Gotcha, I was unsure since `index.ts` had `memoryFs` in lower camel case, that makes sense though.